v1/terraform.go

```diff
@@ -0,0 +1,92 @@
+package atlas
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+)
+
+// TerraformConfigVersion represents a single uploaded version of a
+// Terraform configuration.
+type TerraformConfigVersion struct {
+	Version   int
```

Comment:
This value is automatically calculated by Atlas (I don't think you should POST it). It's an auto-incremementing thing.